app/views/secrets/index.html.erb

```diff
@@ -34,6 +34,9 @@
 <% end %>
 
 <section class="clearfix">
+  <div class="alert alert-info">
+    To use a key in commands, prefix with <b><%= TerminalExecutor::SECRET_PREFIX %></b>
```

Comment:
maybe a bit too visible ... how about a help-test-hover on KEY header ... or somewhere at the bottom of the page